.gitignore

```diff
@@ -1,5 +1,12 @@
+# Define binary
+define
```

Comment:
Good call. The `Makefile` dictates a name for a release build-directory, but a typical build will still spit out a binary with this name. 👍